man/demi_pushto.md

```diff
@@ -16,8 +16,8 @@ int demi_pushto(demi_qtoken_t *qt_out, int sockqd, const demi_sgarray_t *sga, co
 
 ## Description
 
-`demi_pushto()` asynchronously pushes a scatter-gather array to a socket I/O queue. It only works on connection-mode
-sockets.
+`demi_pushto()` asynchronously pushes a scatter-gather array to a socket I/O queue. It only works on connectionless
+sockets (e.g. UDP).
```

Comment:
Since we are improving this, could we change it to match [sendto()](https://pubs.opengroup.org/onlinepubs/9699919799/functions/sendto.html)?

> If the socket is a connectionless-mode socket, the message shall be sent to the address specified by dest_addr if no pre-specified peer address has been set. If a peer address has been pre-specified, either the message shall be sent to the address specified by dest_addr (overriding the pre-specified peer address), or the function shall return -1 and set errno to [EISCONN].

Reply:
Sure, will update the doc to match to sendto  .

Yeah, UDP was just 1 example. But if it works with both connection-mode and connectionless sockets then I'll not mention any example here.

Reply:
Done, updated the doc and added FIXME in code.